docs/tutorials/margins.ipynb

```diff
@@ -82,16 +83,25 @@
     "collapsed": false
    },
    "source": [
-    "Then we load the object catalog, setting the `margin_cache` parameter to the margin catalog we have just loaded"
+    "Here we see the margin catalog that has been generated, in this case using a margin threshold of 10 arcseconds."
```

Comment:
Is there any way for the user to see it is 10 arcseconds? 

After this, it is ok to merge

Reply:
I tried printing the margin size from the catalog_info metadata in the output above, but I agree it's not the most obvious. Do you have any other ideas how to show it?